Sources/Extensions/Foundation/DateExtensions.swift

```diff
@@ -457,6 +457,15 @@ public extension Date {
         return Calendar.current.timeZone
     }
 
+    /// SwifterSwift: Yesterday date.
+    ///
+    ///     var date = Date() // "Oct 3, 2018, 10:57:11"
+    ///     var yesterday = yesterday // "Oct 2, 2018, 10:57:11"
```

Comment:
Why use `var`s in the example if you're not going to mutate them?
Also, this should be `= date.yesterday`